### DISCUSSION TITLE
What are the recommended types for the deprecated types ?

### DISCUSSION THREAD

**Discussion by jinz2014** [intel/llvm#10228] [Q&A] [answered]
It is not clear that these types are deprecated. What are the recommend types ?  Thanks.

```
namespace sycl {
__SYCL_INLINE_VER_NAMESPACE(_V1) {
using byte __SYCL2020_DEPRECATED("use std::byte instead") = std::uint8_t;
using schar __SYCL2020_DEPRECATED("") = signed char;
using uchar __SYCL2020_DEPRECATED("") = unsigned char;
using ushort __SYCL2020_DEPRECATED("") = unsigned short;
using uint __SYCL2020_DEPRECATED("") = unsigned int;
using ulong __SYCL2020_DEPRECATED("") = unsigned long;
using longlong __SYCL2020_DEPRECATED("") = long long;
using ulonglong __SYCL2020_DEPRECATED("") = unsigned long long;
```

**Comment by AlexeySachkov** [comment, marked as answer]
Hi @jinz2014,

Those aliases do not exists in SYCL 2020 specification at all and I'm not even sure that they were required by SYCL 1.2.1. As you can see in their definitions, those aliases contain nothing special and reference standard C++ types. As a replacement, you can either define the same aliases yourself or use corresponding standard C++ types directly
